app/helpers/application_helper.rb

```diff
@@ -3,10 +3,15 @@ module ApplicationHelper
   include SvgHelper
   require "date"
 
-  def fabs_govuk_link_to(text, url, **options)
+  def fabs_govuk_link_to(link_text, url, **options)
     safe_url = safe_url(url)
     external_attrs = external_link_attributes(url)
-    govuk_link_to(text, safe_url, **options.merge(external_attrs))
+    link_text = link_text.html_safe
```

Comment:
calling `.html_safe` tells the templating system that the string does not need further escaping, which isn't true here.